docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
```

Comment:
i see a big difference:

```c#
const  int x = Y + Z + N.C.F; // actual compile time constant.  value is 100% immutable and will be inserted into use sites.
readonly int y = A + B + C(); // executed at runtime.  is mutable. works just like a normal variable, except it can't be overwritten.
```

These are very different semantics.